assem16399/Diva
# feat(Cart) ✨ Implement Cart Feature

## Jira Ticket: [DIVA-23](https://aasemhany.atlassian.net/browse/DIVA-23)

## Description 📖 

Implementing Cart Feature 


## How to Test 📋 
1. Open the app
2. Login or signup your account
3. Check Cart Tab

## Screenshots/Recordings 📷 
![Simulator Screenshot - iPhone 16 Pro - 2024-10-19 at 08 16 40](https://github.com/user-attachments/assets/f553720c-5e20-482a-b3e0-bc3438ea2946)





[DIVA-23]: https://aasemhany.atlassian.net/browse/DIVA-23?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
